tee-worker/identity/client-sdk/packages/client-sdk/src/lib/requests/get-omni-graph.ts

```diff
@@ -0,0 +1,34 @@
+import type { LitentryIdentity, MemberAccount } from '@litentry/parachain-api';
+
+import type { ApiPromise } from '@polkadot/api';
+import { encodeAddress } from '@polkadot/util-crypto';
+
+/**
+ * Retrieve the omni account graph for a given identity.
+ * @param api - Polkadot API instance
+ * @param who - Litentry identity to query
+ * @returns Array of member accounts or null if not found
+ * @throws Error if API calls fail
+ */
+export async function getOmniGraph(
```

Comment:
This getter should probably get the data from the worker (TrustedGetter) as private members are encrypted. But since we are rewriting the id worker in the omni-executor maybe it would be better to wait a bit for this.